pages/docs/tutorials/getting-started/slackbot-websocket-tutorial.md

```diff
@@ -0,0 +1,313 @@
+For example, in `reactionPayload` schema definition, any API message received from this channel must follow the constant value for the "reaction" property which is clearly defined as “heart”.
+
+The const value feature ensures that the data exchanged through your API complies with your specified constants, helping to maintain data integrity and accuracy.
+
+```
+components:
+  messages:
+    reactionListenerMessage:
+      summary: "Action triggered when channel receives a new event of type reaction-added"
+      payload:
+        $ref: '#/components/schemas/reactionPayload'
+    
+    helloListenerMessage:
+      summary: "Action triggered when a successful WebSocket connection is established."
+      payload:
+        $ref: '#/components/schemas/helloPayload'
+
+  schemas:
+    reactionPayload:
+      type: object
+      properties:
+        user:
+          type: string
+          description: ID of the user who performed this event
+        reaction:
+          type: string
+          description: The only reaction that we need is a heart emoji
+          const: "heart"
+        item_user:
+          type: string
+          description: ID of the user that created the original item that has been reacted to
+        item:
+          type: object
+          properties:
+            channel:
+              type: string
+              description: Channel information of original message
+            ts:
+              type: string
+              description: Timestamp information of original message.
+        event_ts:
+          type: string
+          description: Timestamp of reaction
+
+    helloPayload:
+      type: object
+      properties:
+        type:
+          type: string
+          description: A hello string confirming WS connection
+          const: "hello"
+        connection_info:
+          type: object
+          properties:
+            app_id:
+              type: string
+        num_connections:
+          type: integer
+        debug_info:
+          type: object
+          properties:
+            host:
+              type: string
+            started:
+              type: string
+            build_number:
+              type: integer
+            approximate_connection_time:
+              type: integer
+```
+
+
+And there you have it! Putting these blocks together gives you your AsyncAPI document all ready to go.
+
+```
+asyncapi: '3.0.0'
+info:
+  title: MySlackBot API
+  version: '1.0.0'
+  description:  |
+    The MySlackBot App manages popular messages in a workspace by monitoring message reaction data from Slack's Event API.
+    
+servers:
+  ws:
+    host: wss://wss-primary.slack.com/
+    pathname: link/?ticket=13748dac-b866-4ea7-b98e-4fb7895c0a7f&app_id=fe684dfa62159c6ac646beeac31c8f4ef415e4f39c626c2dbd1530e3a690892f
```

Comment:
@KhudaDad414 ideally this query params should not be here and instead should be part of websocket binding

if we add it as binding on the server, will it still work with Glee? does glee understand `query` on binding level? -> https://github.com/asyncapi/bindings/tree/master/websockets#channel-binding-object

Reply:
I had the same question. @VaishnaviNandakumar 
you can read more on it here:
https://asyncapi.slack.com/archives/C0230UAM6R3/p1698241017534699